### PR TITLE
Dev py4jps twa

### PR DESCRIPTION
**Pull request by jb2197**
This PR renames the python wrapper from `py4jps` to `twa`. It's now accessible from PyPI: https://pypi.org/project/twa/0.0.1/

### CODE REVIEW

File: JPS_BASE_LIB/python_wrapper/setup.py [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
+    name='twa',
+    version='0.0.1',
+    author='Jiaru Bai; Daniel Nurkowski',
+    author_email='jb2197@cam.ac.uk; danieln@cmclinnovations.com',
     license='MIT',
-    python_requires='>=3.5',
-    description="Py4jps is a thin Python wrapper for the TheWorldAvatar project.",
-    url="https://github.com/cambridge-cares/TheWorldAvatar/tree/develop/JPS_BASE_LIB/python_wrapper",
+    python_requires='>=3.8',
+    description="twa is a thin Python wrapper for the TheWorldAvatar project.",
```
Comment (sm453) [inline JPS_BASE_LIB/python_wrapper/setup.py:10]: I'd remove the "thin". And also the redundant "the".

File: JPS_BASE_LIB/python_wrapper/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# What is py4jps
+# What is `twa`
 
-`Py4jps` is a thin Python wrapper for the [TheWorldAvatar](https://github.com/cambridge-cares/TheWorldAvatar) project. The code is heavily based on the [py4j](https://www.py4j.org/index.html) package, which enables Python programs running in a Python interpreter to dynamically access Java objects in a Java Virtual Machine.
+`twa` is a thin Python wrapper for the [TheWorldAvatar](https://github.com/cambridge-cares/TheWorldAvatar) project. The code is heavily based on the [py4j](https://www.py4j.org/index.html) package, which enables Python programs running in a Python interpreter to dynamically access Java objects in a Java Virtual Machine. It has a precedent python package, `py4jps`, which is now deprecated.
```
Comment (sm453) [inline JPS_BASE_LIB/python_wrapper/README.md:3]: I'd remove the "thin" and the redundant "the" (in front of "TheWorldAvatar").

**Review by sm453 — CHANGES_REQUESTED** [review]

File: JPS_BASE_LIB/python_wrapper/README.md (hunk 2 above)
Comment (jb2197) [reply JPS_BASE_LIB/python_wrapper/README.md]: Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1149/commits/01a8ee11d9e4caa7e77be0d9fdd9ffd51088d91a

File: JPS_BASE_LIB/python_wrapper/setup.py (hunk 1 above)
Comment (jb2197) [reply JPS_BASE_LIB/python_wrapper/setup.py]: Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1149/commits/01a8ee11d9e4caa7e77be0d9fdd9ffd51088d91a

**Pull request merged**